### DISCUSSION TITLE
Does backlight requires suitable SMBIOS?

### DISCUSSION THREAD

**Discussion by anHiep** [ChefKissInc/NootedRed#317] [Q&A]
I have downloaded the SSDT-PNLF.aml and SSDT-ALS0.aml (because my laptop doesn't have ambient light sensor) from the website. But the screen brightness control is still not work. Some people suggest that I need to change my SMBIOS

**Comment by VisualEhrmanntraut** [comment]
Backlight is auto-enabled only on MacBook SMBIOS. You can force it with AMDBacklight=1

**Reply by anHiep** [reply]
How to do that :v

**Reply by VisualEhrmanntraut** [reply]
...boot-args?

**Reply by anHiep** [reply]
Oh, thanks

**Reply by anHiep** [reply]
It's not work 😭
